### DISCUSSION TITLE
Reset password link without triggering email sending

### DISCUSSION THREAD

**Discussion by sitoftonic** [supabase/supabase#9532] [Questions] [answered]
Is there any way I can get the Reset Password link through a supabase.auth.api method or similar server-side instead of triggering the email sending by default? 
This way I can provide the link into an email template made by myself in an external provider instead of having to work with your system as the email provider. 
Is this possible right now? And if no, are you looking into it anytime soon?

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/reference/javascript/auth-admin-generatelink
I've not used it, but I think this with "recovery link" is what you are looking for.
Than was for v2, there is a similar function for v1.
